examples/import.go

```diff
@@ -1,9 +1,9 @@
-package gifs_go_test
+package main
```

Comment:
So usually the example_test.go file actually contains testable/runnable code that can also be run in godoc.
When the output (if specified in the example_test.go file) produced doesn't match what is wanted in the "// comments" at the end of the function", go test errors out.
This helps us ensure that code that is runnable in the godoc also works when we run `go test`. You can read more about it here https://blog.golang.org/examples